testsuite/performance/README.md

```diff
@@ -49,19 +49,34 @@ Keep reading for more information.
 
 ## Provisioning
 
-### Available provisioners:
+### Provision
 
-- `docker-compose` **Default.** See [`README.docker-compose.md`](README.docker-compose.md) for more details.
+Usage: `mvn verify -P provision[,DEPLOYMENT_PROFILE] [-Dprovisioning.parameters=PARAMETER_SET]`. 
 
-### Provision
+The parameters are loaded from `tests/parameters/provisioning/${provisioning.parameters}.properties` file.
+Individual parameters can be overriden from command line via `-D` params.
+
+To use a custom properties file specify `-Dprovisioning.parameters.properties.file=ABSOLUTE_PATH_TO_FILE` instead of `-Dprovisioning.parameters`.
```

Comment:
Maybe just call it -Dprovisioning.parameters.file (it's shorter), or maybe -Dprovisioning.properties.file.
Although it doesn't really make much of a difference since the whole provisioning is quite complex.

Reply:
Ok, but lets do it for all 3 files to be consistent. 

So maybe like this?
`provisioning.properties` - `provisioning.properties.file`
`dataset.properties` - `dataset.properties.file`
`test.properties` - `test.properties.file`

Or an alternative with "parameters" instead of "properties".

Reply:
That was just a suggestion. Make as you deem appropriate.
The guideline is - the simpler, the better.